app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -31,6 +31,13 @@ def perform(submission_id)
         end
       rescue Common::Exceptions::GatewayTimeout => e
         retryable_error_handler(e)
+      rescue EVSS::DisabilityCompensationForm::ServiceException => e
+        # retry any errors caused by an upstream service from EVSS being unavailable
```

Comment:
Would there be any value in pulling this conditional and comment into a private method, with a yard doc?  i.e. `retry_for_unavailable_external_service!(error)`, or something like that?